### PR TITLE
Planning: initial implemenation of online renderer

### PR DESCRIPTION
**Pull request by JasonZhou404**

### CODE REVIEW

File: modules/planning/learning_based/img_feature_renderer/birdview_img_feature_renderer.cc [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+BirdviewImgFeatureRenderer::BirdviewImgFeatureRenderer() {}
+
+bool BirdviewImgFeatureRenderer::Init(const PlanningSemanticMapConfig& config) {
+  config_ = config;
+
+  // TODO(Jinyun): take map name from upstream and move to conf
+  std::string ROADMAP_IMG_PATH =
+      "/apollo/modules/planning/data/"
+      "sunnyvale_with_two_offices.png";
```
Comment (xiaoxq) [inline modules/planning/learning_based/img_feature_renderer/birdview_img_feature_renderer.cc:35]: Fit in one line.

File: modules/planning/learning_based/img_feature_renderer/birdview_img_feature_renderer.cc [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+namespace apollo {
+namespace planning {
+
+BirdviewImgFeatureRenderer::BirdviewImgFeatureRenderer() {}
+
+bool BirdviewImgFeatureRenderer::Init(const PlanningSemanticMapConfig& config) {
+  config_ = config;
+
+  // TODO(Jinyun): take map name from upstream and move to conf
+  std::string ROADMAP_IMG_PATH =
```
Comment (xiaoxq) [inline modules/planning/learning_based/img_feature_renderer/birdview_img_feature_renderer.cc:33]: static const.

File: modules/planning/learning_based/img_feature_renderer/birdview_img_feature_renderer.cc (hunk 1 above)
Comment (JasonZhou404) [reply modules/planning/learning_based/img_feature_renderer/birdview_img_feature_renderer.cc]: fixed

File: modules/planning/learning_based/img_feature_renderer/birdview_img_feature_renderer.cc (hunk 2 above)
Comment (JasonZhou404) [reply modules/planning/learning_based/img_feature_renderer/birdview_img_feature_renderer.cc]: fixed

**Pull request merged**